apps/astarte_appengine_api/Dockerfile

```
@@ -28,21 +28,26 @@ ADD . .
 RUN mix do compile, release
 
 # Note: it is important to keep Debian versions in sync, or incompatibilities between libcrypto will happen
-FROM debian:bullseye-slim
+FROM debian:bookworm
```

Let's keep the image small (and consistent with the builder one):
```suggestion
FROM debian:bookworm-slim
```